sphinx-doc/testing.rst

```diff
@@ -34,6 +34,7 @@ optional requirements are missing.
 - pytest
 - rowan (optional)
 - CuPy (optional)
+- coxeter (optional)
```

Comment:
Based on the currently failing tests, coxeter is not an optional dependency as advertised here. We should either a) hard code output from coxeter so that it is not even an optional dependency or b) modify the test with skips so that tests will run without coxeter installed.

Reply:
Coxeter is also not just a testing a dependency, but an optional runtime dependency - imported when a user invokes get_ordered_vertices. So it needs to be documented as an optional runtime dependency as well.

Reply:
Why was this resolved?

The documentation has not been updated. `coxeter` is a hard requirement to call the method `md.pair.anis.ALJ.get_ordered_vertices`. Therefore, it should be documented in the `Install prerequisites` section of `BUILDING.rst`.